src/Riok.Mapperly/Descriptors/DescriptorBuilder.cs

```diff
@@ -242,6 +243,11 @@ private void AddUserMapping(IUserMapping mapping, bool ignoreDuplicates, bool na
         _inlineMappings.AddUserMapping(mapping, name);
     }
 
+    private string GetMethodName(IUserMapping mapping)
```

Comment:
This is not (always) the method name, probably use `GetMappingName` or similar?

Reply:
Renamed